build/webpack.common.config.js

```diff
@@ -5,6 +5,10 @@ const { BannerPlugin, DefinePlugin, NormalModuleReplacementPlugin } = require('w
 const license = require('./license');
 const pkg = require('../package.json');
 
+const isLinked = process.env.IS_LINKED === '1';
+const getAnnotationsPath = () =>
```

Comment:
You can use `resolve.symlinks: true`, instead, I think. https://webpack.js.org/configuration/resolve/#resolvesymlinks